api/app/tests/medium/routers/test_users.py

```diff
@@ -22,6 +23,17 @@ def test_create_user():
     assert user1.user_id != ZERO_FILLED_UUID
 
 
+def test_duplicate_email(testdb):
```

Comment:
重複を許すことががテスト名からわかるといいかと
```suggestion
def test_duplicate_email_user_can_be_registered(testdb):
```

Reply:
指摘の通り修正しました。